infrastructure/cdn-in-a-box/README.md

```diff
@@ -1,44 +1,141 @@
-    under the License.
--->
+	Licensed to the Apache Software Foundation (ASF) under one
+	or more contributor license agreements.  See the NOTICE file
+	distributed with this work for additional information
+	regarding copyright ownership.  The ASF licenses this file
+	to you under the Apache License, Version 2.0 (the
+	"License"); you may not use this file except in compliance
+	with the License.  You may obtain a copy of the License at
+
+	  http://www.apache.org/licenses/LICENSE-2.0
 
-CDN In a Box (containerized)
-============================
+	Unless required by applicable law or agreed to in writing,
+	software distributed under the License is distributed on an
+	"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+	KIND, either express or implied.  See the License for the
+	specific language governing permissions and limitations
+	under the License.
+-->
 
-This is intended to simplify the process of creating a "CDN in a box",  easing
+# CDN In a Box (containerized)
+This is intended to simplify the process of creating a "CDN in a box", easing
 the barrier to entry for newcomers as well as providing a way to spin up a
 minimal CDN for full system testing.
 
-For now,  only `traffic_ops` and `traffic_portal` are implemented.  Other
-components will follow as well as details on specific parts of the
-implementation.. 
+## Implemented Components
+As of the time of this writing, Traffic Ops, Traffic Monitor, Traffic Portal and a
+database server for Traffic Ops are all fully implemented. An edge-tier cache,
+mid-tier cache and simple origin are also all implemented, although with limited
+functionality (caches do not respond to update queues/routing, and origin serves
+static HTTP content - likely subject to change in favor of proper video streaming in
+the future). Other components will follow as well as details on specific parts of the
+implementation.
+
+## Setup
+The containers run on docker, and require Docker (tested v17.05.0-ce) and Docker
+Compose (tested v1.9.0) to build and run. On most 'nix systems these can be installed
+via the distribution's package manager under the names `docker-ce` and
+`docker-compose`, respectively (e.g. `sudo yum install docker-ce`).
+
+Each container (except the origin) requires an `.rpm` file to install the Traffic Control
+component for which it is responsible. You can either download these `*.rpm` files or
+create them yourself by using the [`pkg`](../../pkg) script at the root of the
+repository. Copy the `*.rpm`s without any version/architecture information to their
+respective component directories, such that their filenames are as follows:
+
+* `edge/traffic_ops_ort.rpm`
+* `mid/traffic_ops_ort.rpm`
+* `traffic_monitor/traffic_monitor.rpm`
+* `traffic_ops/traffic_ops.rpm`
+* `traffic_portal/traffic_portal.rpm`
+
+Also ensure that the edge- and mid-tier caches have copies of the
+[`traffic_ops/to-access.sh`](./traffic_ops/to-access) script in their own directories.
```

Comment:
rather than doing this,  let's change the context of those services up one level (`context: ..`).   That will require files in the edge, mid, to be referenced in the directory, e.g. `edge/setup.sh` in the Dockerfile,  but won't require this copy step.

Reply:
That's a good idea.